docs/rules/prefer-ember-test-helpers.md

```diff
@@ -0,0 +1,66 @@
+# prefer-ember-test-helpers
+
+This rule ensures the correct Ember test helper is imported when using methods that have a native window counterpart.
+
+There are currently 3 Ember test helper methods that have a native window counterpart:
+
+* blur
+* find
+* focus
+
+If these methods are not properly imported from Ember's test-helpers suite, the native window method version is used instead, and any intended asynchronous functions won't work as intended, which causes tests to fail silently.
+
+See the origin and background of the rule proposal: [eslint-plugin-ember issue 676](https://github.com/ember-cli/eslint-plugin-ember/issues/676)
```

Comment:
Normally we don't want to link to the issue in the documentation. Better for us to ensure all the relevant information is included directly in the doc instead. If someone really wants to, they can seek out the PR/issue on their own.

Reply:
Sure!